zappa/handler.py

```diff
@@ -248,6 +248,7 @@ def import_module_and_get_function(whole_function):
 
     @classmethod
     def lambda_handler(cls, event, context):  # pragma: no cover
+        global handler
```

Comment:
Typically we'd prefer to avoid using `global`.  Could simply avoiding the name collision do the same - something like this?

```python
global_handler = LambdaHandler() if os.environ.get("INSTANTIATE_LAMBDA_HANDLER_ON_IMPORT") else None

...

    @classmethod
    def lambda_handler(cls, event, context):
        handler = global_handler or cls()
```